src/event-handler/index.ts

```diff
@@ -28,7 +28,7 @@ interface EventHandler<TTransformed = unknown> {
   receive(event: EmitterWebhookEvent): Promise<void>;
 }
 
-export function createEventHandler(options: Options<any>): EventHandler {
+export function createEventHandler(options?: Options<any>): EventHandler {
```

Comment:
The `createEventHandler()` argument is optional?

Reply:
I'd say yes